controls/SV-250314.rb

```diff
@@ -40,7 +42,7 @@
       skip 'Control not applicable within a container -- kernel config'
     end
   else
-    describe command('grep sysadm_r /etc/sudoers /etc/sudoers.d/*').stdout.strip do
+    describe command('grep -r sysadm_r /etc/sudoers /etc/sudoers.d').stdout.strip do
       it { should match /TYPE=sysadm_t\s+ROLE=sysadm_r/ }
```

Comment:
Not sure if we need to match the entire output from the check text: `%wheel ALL=(ALL) TYPE=sysadm_t ROLE=sysadm_r ALL`

Reply:
I'd just leave it be, because in theory there can be aliases and other ways of identifying users or groups of users to have admin rights other than using the wheel group.

Reply:
The expected regex is left the way it was. Thanks!